### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by jennahyasin**

### CODE REVIEW

File: src/App.js
Comment (YukioRivera) [inline src/App.js:1]: add a comment about adding route to the app.js to show what changed from teacher's version

File: src/components/AddStudent.js
Comment (YukioRivera) [inline src/components/AddStudent.js:1]: add some comments about what each section is doing 
maybe include some points about how its interacting with the backend 
overall code looks good

File: src/components/Semester.js
Comment (YukioRivera) [inline src/components/Semester.js:1]: nice customization of the button

**Review by YukioRivera — COMMENTED** [review]
good code but just add some comments

**Comment by jennahyasin** [comment]
thanks for the feedback Yukio! I just added some comments to my code to make it better understood.

**Pull request merged**